src/temporal/notifications/utils.ts

```diff
@@ -38,12 +46,16 @@ export const runReminderWorkflow = async (params: ReminderWorkflowParams) => {
 export const cancelReminderWorkflow = async (
   params: ReminderWorkflowParams,
 ) => {
-  const client = await getTemporalClient();
   const workflowId = getReminderWorkflowId(params);
-  const handle = client.workflow.getHandle(workflowId);
-  const description = await handle.describe();
+  const handle = await getWorkflowHandle(workflowId);
+
+  if (!handle) {
+    return;
+  }
```

Comment:
Maybe will be more visible when tests are added, but in what case the handle is missing?

Reply:
It is a third-party package, so I was just assuming the worst that it could be null or undefined.

Reply:
Assuming worst is fine but returning early does not fix it, I think its important to understand if and when it can be nullish and if we need to handle it.

Reply:
But lets see the test cases

Reply:
In this case, it is canceling a workflow, so if it does not exist, we can just ignore the process because we will have nothing to cancel.